src/ERC721Drop.sol

```diff
@@ -732,21 +756,29 @@ contract ERC721Drop is
         _mintSupplyRoyalty(quantity);
         _requireCanMintQuantity(quantity);
 
-        address msgSender = _msgSender();
-
-        _requireMerkleApproval(msgSender, maxQuantity, pricePerToken, merkleProof);
-
-        _requireCanPurchasePresale(msgSender, quantity, maxQuantity);
-
-        _handleRewards(msg.value, quantity, pricePerToken, config.fundsRecipient != address(0) ? config.fundsRecipient : address(this), createReferral, mintReferral);
-
-        _mintNFTs(msgSender, quantity);
-
-        uint256 firstMintedTokenId = _lastMintedTokenId() - quantity;
-
-        _emitSaleEvents(msgSender, msgSender, quantity, pricePerToken, firstMintedTokenId, comment);
-
-        return firstMintedTokenId;
+<<<<<<< HEAD
```

Comment:
broken rebase :(

Reply:
was trying to get this diff better, didn't work, ignore for now